### PR TITLE
Add nation and HHS to CHC

### PR DESCRIPTION
**Pull request by chinandrew**
### Description
**Merge after #610**

Add HHS and nation to Change. 

### Changelog
Itemize code/test/documentation changes and files added/removed.
- Update conditional statements to map to nation and hhs
- Update tests to run the hhs and nation cases

### Fixes 
- n/a

### CODE REVIEW

File: changehc/delphi_changehc/update_sensor.py [hunk 1]
```diff
@@ -158,10 +156,10 @@ def geo_reindex(self, data):
...
         elif geo == "state":
             data_frame = gmpr.replace_geocode(data, "fips", "state_id", new_col="state")
-        elif geo == "msa":
-            data_frame = gmpr.replace_geocode(data, "fips", "msa")
-        elif geo == "hrr":
-            data_frame = gmpr.replace_geocode(data, "fips", "hrr")
+        elif geo in ["msa", "hrr", "hhs"]:
+            data_frame = gmpr.replace_geocode(data, "fips", geo)
+        else:
+            data_frame = gmpr.replace_geocode(data, "fips", "nation")
```
Comment (rumackaaron) [inline changehc/delphi_changehc/update_sensor.py:162]:
```suggestion
        else:
            data_frame = gmpr.replace_geocode(data, "fips", geo)
```
Any reason to not include "nation" in the catch-all?

**Comment by rumackaaron** [comment]
We'd also have to change `params.json` in `ansible/templates/changehc-params-prod.json.j2`

File: changehc/delphi_changehc/update_sensor.py (hunk 1 above)
Comment (chinandrew) [reply changehc/delphi_changehc/update_sensor.py:162]: Nope, good catch.

**Comment by chinandrew** [comment]
> We'd also have to change `params.json` in `ansible/templates/changehc-params-prod.json.j2`

Ah thanks for pointing this out, I've added that change

**Review by rumackaaron — APPROVED** [review]

**Pull request merged**